guides/release/components/customizing-a-components-element.md

```diff
@@ -33,6 +34,36 @@ export default Component.extend({
 </ul>
 ```
 
+Note: It is not recommended to use null tagName .
```

Comment:
```suggestion
Note: It is not recommended to use `tagName: ''`.
```

Reply:
Fixing this should also fix the failing test, since the spellchecker ignores words that are wrapped in backticks.